OGC-Core/IncorrectNumberOfPoints.class.st

```diff
@@ -0,0 +1,39 @@
+"
+Error used when too many points are given to instanciate a Line.
+"
+Class {
+	#name : #IncorrectNumberOfPoints,

```

Comment:
Please consider to make all classes have the same prefix OGC

Reply:
Indeed, I was inspired by Collections package provided directly in Pharo which doesn't include prefix
Now, I renamed my error classes with "OGCError" prefix

Reply:
I don't want to nitpick just to give some details. If you use Error in the name it is usually added at the end. So it would be a OGCIncorrectNumberOfPointsError